demo/channel.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by avim on 6/27/16.
+ */
+
+import React from 'ReactOverTheWire';
+import ReactOverTheWireDOM from 'ReactOverTheWireDOM';
+import recursion from './recursion/app.jsx';
+import dbmonster from './dbmonster/app.jsx';
+import drag from './drag/app.jsx';
+import masonry from './masonry/app.jsx';
+import todo from './todo/app.jsx';
+import nativeExtensions from './nativeExtensions';
+
+const demoApps = {recursion,dbmonster,drag,masonry,todo};
+
+function channelFacade(port, title) {
```

Comment:
👍 This is a good idea. Angular2's web worker implementation does something very similar - they also have a pretty robust message bus.